src/tike/ptycho/probe.py

```diff
@@ -138,6 +138,20 @@ class ProbeOptions:
     hard constraint.
     """
 
+    median_filter_abs_probe: bool = False
+    """Binary switch on whether to apply a median filter to absolute value of 
+    each shared probe mode.
+    """
```

Comment:
Aren't `median_filter_abs_probe` and `median_filter_abs_probe_period` redundant? For example, you could set the filter period to 0 or something large like 2^32-1 to disable it?

Reply:
I've removed `median_filter_abs_probe_period` altogether; see other comment below.